autorest/codegen/__init__.py

```diff
@@ -71,6 +71,9 @@ def _validate_code_model_options(options: Dict[str, Any]) -> None:
     if options["basic_setup_py"] and not options["package_version"]:
```

Comment:
can you update the changelog, bumping minor version? thanks!

Reply:
done